const Exception = () => {
    return (
        <div>
            异常详情
        </div>
    )
}
export default Exception